### PR TITLE
page-form - integer field

### PR DESCRIPTION
**Pull request by collin**
PR for trello

### CODE REVIEW

**Comment by collin** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;NiNrUCXu&#x2F;62-integer"> integer</a>

**Comment by collin** [comment]
@kand I think this field is ready.

File: app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-integer/dynamic-content-form-field-integer.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        },
+        templateUrl: CmsConfig.buildComponentPath(
+          'dynamic-content',
+          'dynamic-content-form',
+          'dynamic-content-form-field',
+          'dynamic-content-form-field-integer',
+          'dynamic-content-form-field-integer.html'
+        ),
+        link: function (scope, element, attr, ctrls) {
+          var formField = ctrls[1][scope.name];
```
Comment (kand) [inline app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-integer/dynamic-content-form-field-integer.js]: Given that the input is already type `number` I believe this validator comes for free out of the box: https://code.angularjs.org/1.3.19/docs/api/ng/input/input%5Bnumber%5D

File: app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-integer/dynamic-content-form-field-integer.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          formField.$validators.integer = function (modelValue) {
```
Comment (kand) [inline app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-integer/dynamic-content-form-field-integer.js]: Given that the input is already type `number` I believe this validator comes for free out of the box: https://code.angularjs.org/1.3.19/docs/api/ng/input/input%5Bnumber%5D

Available in `$error` as `number`.

**Comment by kand** [comment]
👍

**Pull request merged**